remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/LifecycleManager.java

```diff
@@ -113,6 +115,7 @@ private void initProtobufMetadataManager(GlobalConfiguration globalCfg, GlobalCo
       SerializationContext serCtx = protobufMetadataManager.getSerializationContext();
       ClassLoader classLoader = globalCfg.classLoader();
       processProtostreamSerializationContextInitializers(classLoader, serCtx);
+      processSerializationContextInitializer(globalCfg.serialization().contextInitializer(), serCtx);
```

Comment:
I believe a null check is needed here. A CI run has a NPE https://ci.infinispan.org/job/Infinispan/job/PR-7205/6/consoleFull#1709264435ea5a4324-14c7-4883-b210-e33383209aa7   at org.infinispan.query.remote.impl.LifecycleManager.processSerializationContextInitializer(LifecycleManager.java:142)

Reply:
You're right! That's actually why I put the code into a method, but I forgot to add the check/return :upside_down_face:. Added